includes/api/ApiQueryUsers.php

```diff
@@ -161,11 +161,13 @@ public function execute() {
 
 				/* Wikia change begin - SUS-92 */
 				if ( isset( $this->prop['blockinfo'] ) || isset( $this->prop['localblockinfo'] ) ) {
-					$isGlobalBlockCheck = !isset( $this->prop['localblockinfo'] );
-					$isBlocked = $user->isBlocked( true, false, $isGlobalBlockCheck );
+					// SUS-1456: Only perform lookup for global blocks if user is allowed to view this info
+					$canViewGlobalBlocks = $this->getUser()->isAllowed( 'phalanx' );
+					$checkGlobalBlocks = $canViewGlobalBlocks && !isset( $this->prop['localblockinfo'] );
+					$isBlocked = $user->isBlocked( true /* use slave DB */, false /* don't log in phalanx */, $checkGlobalBlocks );
```

Comment:
Let's get in touch with Platform / Services team on this one. AFAIR they use this API end-point in Pandora to check blocks.

@jcellary / @sqreek